client/checkout/upe-styles/index.js

```diff
@@ -234,27 +257,23 @@ const hiddenElementsForUPE = {
 	},
 };
 
-export const getFieldStyles = ( selector, upeElement, focus = false ) => {
+export const getFieldStyles = ( selector, upeElement ) => {
 	if ( ! document.querySelector( selector ) ) {
 		return {};
 	}
 
 	const validProperties = upeRestrictedProperties[ upeElement ];
 
 	const elem = document.querySelector( selector );
-	if ( focus ) {
-		elem.focus();
-	}
```

Comment:
This caused the page to jump upwards to the hidden input on the blocks checkout when the focus state is triggered. I found that the computed styles for the focus state were not tremendously significant, so I removed them entirely. We can depend on Stripe's defaults for the focus state instead. 🤷